homeassistant/components/doorbird/__init__.py

```diff
@@ -137,16 +94,126 @@ def _reset_device_favorites_handler(event):
             for favorite_id in favorites[favorite_type]:
                 doorstation.device.delete_favorite(favorite_type, favorite_id)
 
-    hass.bus.listen(RESET_DEVICE_FAVORITES, _reset_device_favorites_handler)
+    hass.bus.async_listen(RESET_DEVICE_FAVORITES, _reset_device_favorites_handler)
```

Comment:
this shouldn't be an event. This should be done with the dispatcher. Can be in a future PR.

Reply:
Yup, more cleanup is needed here.  Another PR for sure as this one is already getting longer than I had expected.